deprecations.js

```diff
@@ -18,6 +18,10 @@ const versionDeprecations = {
     {
       selectors: [':-ms-input-placeholder'],
       message: 'Browserslist update to match github has removed the need for this pseudoselector'
+    },
+    {
+      selectors: ['.btn-outline-blue'],
+      message: `This selector is not available in Primer CSS 17.0.0. Please refer to the documentation.`
```

Comment:
If a class is only removed from the docs, then adding the class to `deprecations.js` shouldn't be needed. I think the deprecation script only scans the `src/`.